tests/test_pin_job.py

```diff
@@ -0,0 +1,36 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+from pages.treeherder import TreeherderPage
+
+class TestUnclassifiedJobs:
+
+    def test_pin_next_job(self, base_url, selenium):
+        """ Open treeherder page, select next job and pin it"""
+        page = TreeherderPage(base_url, selenium).open()
+        current_job_title = page.result.select_next_job()
```

Comment:
The page is split into several result sets, each has one or more jobs. I think the regions in the page object model should reflect that, so the test would be page.result_set[0].jobs[0].click() to select the first job of the first result set. I know this is a little different from my previous comment, as I'm taking into account multiple result sets. I think a `select_next_job` function is still valuable, but it feels like it should be a top level function in the `TreeherderPage` rather than within a page region.